code/modules/spells/spell_types/pointed/_pointed.dm

```diff
@@ -16,8 +16,8 @@
 	var/deactive_msg
 	/// The casting range of our spell
 	var/cast_range = 7
-	/// Variable dictating if the spell will use turf based aim assist
-	var/aim_assist = TRUE
+	/// List variable which contains atom types used as backup targets if the spell misses. Default humans and living, but can be anything
+	var/list/aim_assist_list = list(/mob/living, /mob/living/carbon/human)
```

Comment:
The old method of aim assist prioritized humans over living mobs, that behavior should be maintained here - 
perhaps by having the order of the list determine priority


(Actually currently it natively prioritizes based on list order, so all you need to do is re-arrange this to be `human -> living`)